app/src/androidTest/java/ch/epfl/sweng/vanjel/LoginActivityTest.java

```diff
@@ -41,80 +43,57 @@
     public final ActivityTestRule<LoginActivity> ActivityRule =
             new ActivityTestRule<>(LoginActivity.class);
 
-    @Before
-    public void init() {
-        FirebaseAuthCustomBackend.setNullUser(true);
-        ActivityRule.finishActivity();
-        Intent i = new Intent();
-        ActivityRule.launchActivity(i);
+    @BeforeClass
+    public static void setUp() {
```

Comment:
why did you remove @Before that were there in order to set the user as null with the mock? 
Depending on the previous test suite, it can set it with a patient or a doctor

Reply:
well sorry, didn't see the setupNoExtras (even If I think having too much booleans in the constructor can be not as readable as multiple lines that explicitly shows what variables are set)

Reply:
It's a helper functions so I guess it is not very important, see with it's author @ljoss17